### PR TITLE
Amp viafoura

### PR DESCRIPTION
**Pull request by dougalg**
https://github.com/ampproject/amphtml/issues/2442

### CODE REVIEW

**Comment by dougalg** [comment]
I signed it!

**Comment by Tckool** [comment]
I am okay with these commits being contributed to Google

**Comment by dvoytenko** [comment]
Thanks a lot, @dougalg ! I'll start the review of this pull request. In the meantime, could you please check on CLA? It looks like there are couple of contributors in the commits. Could you please squash them and leave one author? Please also make sure that the email you use in the commit is the same you use for CLA. Thank again!

**Comment by rudygalfi** [comment]
cc @ericlindley-g

**Comment by dougalg** [comment]
Thanks @dvoytenko . Will do.

**Comment by Tckool** [comment]
I signed it!

File: extensions/amp-viafoura/0.1/amp-viafoura.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (erwinmombay) [inline extensions/amp-viafoura/0.1/amp-viafoura.js]: 2016 here and every other new file

**Comment by mrjoro** [comment]
Thank you for going through the effort of creating this PR.  It hasn't been updated in a while, so we're going to close it for now.  If you are still actively working on this PR or find yourself blocked on something/someone please reopen the PR with a comment giving the latest status.

**Pull request closed without merging**